base/core/src/message_to_journeys.cc

```diff
@@ -47,6 +47,14 @@ journey::transport to_transport(Walk const& walk, uint16_t duration) {
   return t;
 }
 
+static uint32_t to_color(std::string_view const color_str) {
+  if (color_str.size() != 6) {
+    return 0;
+  }
+  uint32_t const c = 0xFF000000U | static_cast<uint32_t>(std::strtol(color_str.data(), nullptr, 16));
```

Comment:
Can you please convert this to [AAA style](https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/)? I'm not sure if that's a cast. If it is, `static_cast` on the right side (and just `auto const` on the left) would make it more obvious.